configure.ac

```diff
@@ -35,9 +35,10 @@ AC_INIT([EOS Event Recorder Daemon], [0.0.0],
 # Verify that the source directory can be found
 AC_CONFIG_SRCDIR([daemon/emer-daemon.h])
 # Initialize Automake: enable all warnings and do not insist on GNU standards.
-# no-portability suppresses warnings about syntax specific to GNU make
+# no-portability suppresses warnings about syntax specific to GNU make.
 # parallel-tests specifies that we use the new parallel-running test harness.
 # Unlike serial-tests, this option is accepted by Automake 1.11.
+# subdir-objects mirrors the source tree in the build directory.
```

Comment:
In Automake 2.0 subdir-objects will always be turned on, with no option to turn it off, as far as I know. The warning telling you to enable it is so that you fix any breakage now instead of having it happen by surprise when you upgrade to 2.0.